sdk/cosmos/azure-cosmos/azure/cosmos/aio/_container.py

```diff
@@ -435,6 +436,65 @@ def read_all_items(
         )
         return items
 
+    @distributed_trace_async
+    async def read_many_items(
+            self,
+            items: List[Tuple[str, PartitionKeyType]],
```

Comment:
```suggestion
            items: Sequence[Tuple[str, PartitionKeyType]],
```
I think typing this as `Sequence` might be a bit more user-friendly.